src/getHumanAge.test.js

```diff
@@ -7,4 +7,28 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should work with zeros', () => {
+    const result = getHumanAge(0, 0);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should work correct with young animals', () => {
+    const result = getHumanAge(10, 10);
+
+    expect(result).toEqual([0, 0]);
```

Comment:
The expected result for young animals seems incorrect. If the function is supposed to calculate human age equivalents, verify the logic and expected output for young animals.